{"name": "test_delete_some_contact", "status": "failed", "statusDetails": {"message": "AssertionError: assert (54 - 1) == 54\n +  where 54 = len([108:lastnamebcMpfFY5X:firstnameA1t, 109:lastnameF7:firstname, 116:lastnametuQXwFRKTMp0x:firstnamejQ2a56, 118:lastnamebcMpfFY5X:firstnameA1t, 131:lastnamenatEAxcif:firstnamewAMcl, 132:lastnamebcMpfFY5X:firstnameA1t, ...])\n +  and   54 = len([108:lastnamebcMpfFY5X:firstnameA1t, 109:lastnameF7:firstname, 116:lastnametuQXwFRKTMp0x:firstnamejQ2a56, 118:lastnamebcMpfFY5X:firstnameA1t, 131:lastnamenatEAxcif:firstnamewAMcl, 132:lastnamebcMpfFY5X:firstnameA1t, ...])", "trace": "app = <fixture.application.Application object at 0x0000026AE1171580>, db = <fixture.db.DbFixture object at 0x0000026AE11715E0>, check_ui = False\n\n    def test_delete_some_contact(app, db, check_ui):\n        if len(db.get_contact_list()) == 0:\n            app.contact.create(Contact(firstname=\"test\"))\n        with allure.step(\"Given an contact list\"):\n            old_contacts = db.get_contact_list()\n        with allure.step(\"Given a selected contact\"):\n            contact = random.choice(old_contacts)\n        with allure.step(\"When I delete the contact\"):\n            app.contact.delete_contact_by_id(contact.id)\n        with allure.step(\"Then the new contact list is equal to the old list with the deleted contact\"):\n            new_contacts = db.get_contact_list()\n>           assert len(old_contacts) - 1 == len(new_contacts)\nE           assert (54 - 1) == 54\nE            +  where 54 = len([108:lastnamebcMpfFY5X:firstnameA1t, 109:lastnameF7:firstname, 116:lastnametuQXwFRKTMp0x:firstnamejQ2a56, 118:lastnamebcMpfFY5X:firstnameA1t, 131:lastnamenatEAxcif:firstnamewAMcl, 132:lastnamebcMpfFY5X:firstnameA1t, ...])\nE            +  and   54 = len([108:lastnamebcMpfFY5X:firstnameA1t, 109:lastnameF7:firstname, 116:lastnametuQXwFRKTMp0x:firstnamejQ2a56, 118:lastnamebcMpfFY5X:firstnameA1t, 131:lastnamenatEAxcif:firstnamewAMcl, 132:lastnamebcMpfFY5X:firstnameA1t, ...])\n\ntests_for_allure\\test_delete_contact.py:17: AssertionError"}, "steps": [{"name": "Given an contact list", "status": "passed", "start": 1642637128400, "stop": 1642637128402}, {"name": "Given a selected contact", "status": "passed", "start": 1642637128402, "stop": 1642637128402}, {"name": "When I delete the contact", "status": "passed", "start": 1642637128402, "stop": 1642637129612}, {"name": "Then the new contact list is equal to the old list with the deleted contact", "status": "failed", "statusDetails": {"message": "AssertionError: assert (54 - 1) == 54\n +  where 54 = len([108:lastnamebcMpfFY5X:firstnameA1t, 109:lastnameF7:firstname, 116:lastnametuQXwFRKTMp0x:firstnamejQ2a56, 118:lastnamebcMpfFY5X:firstnameA1t, 131:lastnamenatEAxcif:firstnamewAMcl, 132:lastnamebcMpfFY5X:firstnameA1t, ...])\n +  and   54 = len([108:lastnamebcMpfFY5X:firstnameA1t, 109:lastnameF7:firstname, 116:lastnametuQXwFRKTMp0x:firstnamejQ2a56, 118:lastnamebcMpfFY5X:firstnameA1t, 131:lastnamenatEAxcif:firstnamewAMcl, 132:lastnamebcMpfFY5X:firstnameA1t, ...])\n", "trace": "  File \"C:\\Users\\Vafelence\\PycharmProjects\\python_autotests\\tests_for_allure\\test_delete_contact.py\", line 17, in test_delete_some_contact\n    assert len(old_contacts) - 1 == len(new_contacts)\n"}, "start": 1642637129612, "stop": 1642637129615}], "start": 1642637128397, "stop": 1642637129615, "uuid": "76765512-c425-47c1-be61-e874b3debece", "historyId": "c5fc2405f7060ce5f18de099bd5b5ed7", "testCaseId": "34554150551dbead753da9863589e595", "fullName": "tests_for_allure.test_delete_contact#test_delete_some_contact", "labels": [{"name": "parentSuite", "value": "tests_for_allure"}, {"name": "suite", "value": "test_delete_contact"}, {"name": "host", "value": "VafelencePC"}, {"name": "thread", "value": "11448-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests_for_allure.test_delete_contact"}]}